agents/updater/src/submit.rs

```diff
@@ -54,12 +60,20 @@ impl UpdateSubmitter {
 
                     // Submit update and let the home indexer pick up the
                     // update once it is confirmed state in the chain
-                    self.home.update(&signed).await?;
+                    let tx = self.home.update(&signed).await?;
 
                     self.submitted_update_count.inc();
 
-                    // continue from local state
+                    // Continue from local state
                     committed_root = signed.update.new_root;
+
+                    // Sleep for finality x blocktime seconds to wait for
+                    // timelag reader to catch up
+                    info!(
+                        tx_hash = ?tx.txid,
+                        "Submitted update with tx hash {:?}. Sleeping for {} seconds before next tx submission.", tx.txid, self.finalization_seconds,
```

Comment:
use `sleep = self.finalization_seconds` for these messages instead of string formatting